doc/services/pm/device.rst

```diff
@@ -67,12 +67,6 @@ option for device power management.
     a low power state only if no device is in the middle of a hardware
     transaction that cannot be interrupted.
 
-.. note::
-
-    This type of device power management is disabled when
```

Comment:
`CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE` is a kind of `CONFIG_PM_DEVICE=n`, so if you're using it, you don't want `CONFIG_PM_DEVICE`, just that you can't because device runtime depends on it. When using both, if it's the case, PM_DEVICE will still skip devices using runtime PM.